pages/contribute/editorial_board_guide.md

```diff
@@ -230,3 +230,11 @@ related_pages:
 ### Page ID
 
 To find out what the `page_id` of an RDMkit page is, please check its metadata attribute `page_id` at the top of the markdown file or the [Website overview page](website_overview).
+
+
+### Links between RDMkit and FAIR Cookbook
```

Comment:
```suggestion
### Linking from RDMkit to FAIR Cookbook
```

Since this is the RDMkit editorial process I guess we can say it directly this way?